framework/infrastructure/gcp/api.py

```diff
@@ -175,6 +176,26 @@ def secrets(version: str):
 
         raise NotImplementedError(f"Secret Manager {version} not supported")
 
+    @functools.lru_cache(None)
+    def monitoring_metric_service(self, version: str):
+        """Monitoring API Metric Service.
+
+        Manages metric descriptors, monitored resource descriptors,
+        and time series data.
+
+        https://cloud.google.com/python/docs/reference/monitoring/latest/google.cloud.monitoring_v3.services.metric_service.MetricServiceClient
```

Comment:
[Minor] Don't we need some spaces for this line?

Reply:
Sorry, not sure what you mean here.

Reply:
NVM, looks like it's a github UI thing, when reviewing in `Files changed` tab, this line don't have any space in front of it.